include/coruja/container/view/container.hpp

```diff
@@ -15,10 +15,9 @@
 
 namespace coruja { namespace view {
 
-template<typename ObservableErasableRange>
```

Comment:
There is a common practice in C++ prior to the adoption of concepts in the language to name the template parameters with the concept name. It's important to inform the programmer about the requirements. As you know, an `ObservableErasableRange` is a refinement of a `Range` or (`Rng`).